llvm/test/Transforms/LoopVectorize/RISCV/gather-scatter-cost.ll

```diff
@@ -2,6 +2,9 @@
 ; RUN: opt < %s -passes=loop-vectorize -mtriple riscv64 -mattr=+rva23u64 -S | FileCheck %s -check-prefixes=CHECK,RVA23
 ; RUN: opt < %s -passes=loop-vectorize -mtriple riscv64 -mattr=+rva23u64,+zvl1024b -S | FileCheck %s -check-prefixes=CHECK,RVA23ZVL1024B
 
+%t0 = type { ptr, %t1, ptr }
+%t1 = type { i32, i32, i32, i32, i32, ptr, ptr, i32, i32, i32, i32, i32, ptr, ptr, i32, i32, i32, i32, i32, i32 }
+
```

Comment:
Do we need these custom types? Can we reproduce the crash with just a plain gep into a `i32`?